src/libs/blueprint/conduit_blueprint_mesh_examples.hpp

```diff
@@ -101,18 +101,6 @@ namespace examples
                                      conduit::index_t nz,
                                      conduit::Node &res);
 
-    /// Generates a rectilinear grid with a scalar field that
-    /// visualizes the julia set (https://en.wikipedia.org/wiki/Julia_set)
-    void CONDUIT_BLUEPRINT_API julia(conduit::index_t nx,
-                                     conduit::index_t ny,
-                                     conduit::float64 x_min,
-                                     conduit::float64 x_max,
-                                     conduit::float64 y_min,
-                                     conduit::float64 y_max,
-                                     conduit::float64 c_re,
-                                     conduit::float64 c_im,
-                                     conduit::Node &res);
-
     /// Generates a multi-domain fibonacci estimation of a golden spiral.
```

Comment:
can we include the new julia header here - to all folks to access all of the examples via one header include?

Reply:
I followed the example from the way the mesh examples are currently included in the test. The `conduit_blueprint.hpp` file contains all includes.